Shell/Skolem.hpp

```diff
@@ -107,8 +107,8 @@ class Skolem
   DHMap<unsigned,TermList> _varSorts;
 
   // for some heuristic evaluations after we are done
-  Stack<unsigned> _introducedSkolemSyms;
-
+  Stack<pair<bool, unsigned>> _introducedSkolemSyms;
```

Comment:
Will we ever introduce a Skolem predicate? I assume FOOL takes care of this.

Reply:
That's right